### ISSUE TITLE
Plot cursor shows wrong value when unit prefixes are used

### ISSUE THREAD

**Issue by christiankral** [OpenModelica/OpenModelica#9754]
### Description

When ploting a quanitity with unit prefixes, the currsor value is not correct.

### Steps to Reproduce

1. Simulate `Modelica.Electrical.Machines.Examples.InductionMachines.IMC_DOL`
2. Plot `aimc.powerBalance.powerStator`

![image](https://user-images.githubusercontent.com/4184218/203822503-5e370618-50f0-45aa-abcd-b8a9235ba34c.png)

This cursor shows:
**(kW)**
**Value: 67016.1**

But the real value is 67.0161 kW =  67016.1 W which can be double checked from the axis scaling. 

Displaying the W value but the kW unit is not correct and misleading.

### Expected Behavior

The cursor shall show 
**(kW)**
**Value: 67.0161**

### Version and OS

```
Connected to OpenModelica 1.21.0~dev-43-g20d2d19
Connected to OMSimulator Linux Mint 20.3
```

**Comment by christiankral** [comment]
@casella Please correct me if I am wrong: I think that displaying the correct simulation result is a very high priority for a stable release version. On my opinion the fix of this issue shall be included in the next stable release version.

**Comment by casella** [comment]
Oops, we missed that, good that you spotted it out 😅. @christiankral you are absolutely right. I guess one could easily figure out that what is displayed is 67015 W (not MW), but it's definitely not nice.

@adeas31 this looks like a quick fix, could you sneak it in maintenance/v1.20 tomorrow?

**Comment by bilderbuchi** [comment]
did this PR by chance accidentally also fix #8478, which is the same class of bug (wrong unit vs. values in plots).

**Comment by adeas31** [comment]
Ported this to 1.20 #9765
